### ISSUE TITLE
Indicate what mailing list a visible email is shown from

### ISSUE THREAD

**Issue by johnament** [apache/incubator-ponymail#211]
When searching for an email, if its included in multiple email lists, the individual message view does not indicate what list its from.  This makes linking to them for archival purposes, e.g. board emails, hard to follow and confirm.

**Comment by sebbASF** [comment]
Works OK for me, except in 'flat' View mode, when the Sender is shown rather than the list name

**Comment by johnament** [comment]
@sebbASF example link? you can send it direct if you prefer..

**Comment by sebbASF** [comment]
The following works for me:

https://lists.apache.org/list.html?*@tomcat.apache.org:lte=1M:%22webapp%20log%20in%20tomcat%22

This is a search across multiple lists; I get two results - in dev@ and user@

**Comment by sebbASF** [comment]
![screen shot 2016-10-26 at 01 36 03](https://cloud.githubusercontent.com/assets/16689231/19709228/e34ef83a-9b1c-11e6-83c4-5185647de264.png)

**Comment by sebbASF** [comment]
It looks as though the displayed items depend on what the search box shows as well as the mode.
The list name is only shown if the search box shows a wild-card list. It also looks as though the wildcard list from the URL is not propagated into the search box.
Note: the URL above no longer works because it uses a relative time. Replace lte=1M with lte=nM as needed.

Also in flat mode it looks as if the list name is shown on the first e-mail of a thread only, and not when the Default theme is used - in which case the sender only is displayed. Whereas for other modes the listname is shown in all themes (if the search box is wildcarded)
